setup.py

```diff
@@ -39,6 +39,7 @@
         'taca.subcommands': [
             'storage = taca.storage.cli:storage',
             'analysis = taca.analysis.cli:analysis',
+            'bioinfo = taca.utils.cli:bioinfo',
```

Comment:
why not create `bioinfo` directory ? like `storage` and `analysis`
(my **OCD** tingles :stuck_out_tongue: )